File: mac/Sources/ClaudeToolbarCore/Settings/MascotMode.swift
import Foundation

public enum MascotMode {
    public static let full = "full"
    public static let hover = "hover"
    public static let off = "off"
    public static let all = [full, hover, off]

    public static func normalize(_ value: String?) -> String {
        let key = value?.trimmingCharacters(in: .whitespacesAndNewlines).lowercased() ?? ""
        return all.contains(key) ? key : full
    }
}

File: src/ClaudeToolbar.Core/Settings/MascotMode.cs
namespace ClaudeToolbar.Core.Settings;

public static class MascotMode
{
    public const string Full = "full";
    public const string Hover = "hover";
    public const string Off = "off";

    public static IReadOnlyList<string> All { get; } = [Full, Hover, Off];

    public static string Normalize(string? value)
    {
        var key = value?.Trim().ToLowerInvariant() ?? string.Empty;
        return All.Contains(key) ? key : Full;
    }
}
